features/HomeTab/FlatOffer/Footer/Footer.tsx

```diff
@@ -0,0 +1,45 @@
+import { useContext } from "react";
+import { View } from "react-native";
+import { useTheme, Button } from "react-native-paper";
+
+import SettingsContext from "../../../../contexts/SettingsContext";
+import getPriceWithCurrency from "../../../../preferences/currencies";
+import translations from "../../../../preferences/translations";
+
+interface HeaderProp {
+  navigation: any;
+  price: any;
+}
+
+const Footer = ({ navigation, price }: HeaderProp) => {
+  const theme = useTheme();
+  const { settings } = useContext(SettingsContext);
+
+  return (
+    <View
+      style={{
+        alignItems: "center",
+        flexDirection: "row",
+        borderTopWidth: 1,
+        borderTopColor: theme.colors.backdrop,
+        marginTop: 10,
+        padding: 10,
+      }}>
+      <Button style={{ flex: 1, alignItems: "flex-start" }} labelStyle={theme.fonts.titleLarge}>
+        {getPriceWithCurrency(price, settings.currency, 0)}/{translations.NIGHT[settings.language]}
```

Comment:
Change `$1000/night` -> `$1000 per night`. It's not wrong, but would be lovely to keep consistency over whole app (at the flat offer list there is $$$ per night).